### DISCUSSION TITLE
Android 5-8.1 End of Support soon

### DISCUSSION THREAD

**Discussion by Mygod** [Mygod/VPNHotspot#452] [General]
This is an announcement that the next major update (v2.16) will not support Android 5-8.1. This is due to a subtle bug discovered in the latest release (v2.15.4) as well as general lack of donations from users running on these older platforms. Furthermore, there has been no core features being added for Android 5-11 ever since v2.13.0 (adding auto start services support) released in 2021, so we expect this change to have minimal impact.

Moving forward, [v2.15](https://github.com/Mygod/VPNHotspot/tree/lollipop) is not expected to receive any update unless for critical bug fixes, which is unlikely. However, Android 5-8.1 users can continue to run v2.15 on their devices, which honestly should work just fine.

Fun fact: [v1.0.0 was written for Android 5-8.1](https://github.com/Mygod/VPNHotspot/blob/1.0.0/mobile/build.gradle). Feels old man.

**Comment by cuynu** [comment]
Android 5 - 8.1 are all outdated, They are all over 6 years old. Abandonment is right
